frontend/src/scenes/organization/Settings/Members.tsx

```diff
@@ -227,13 +228,13 @@ export function Members({ user }: { user: UserType }): JSX.Element {
             dataIndex: 'actions',
             key: 'actions',
             align: 'center',
-            render: ActionsComponent,
+            render: (_, member) => ActionsComponent(member as OrganizationMemberType),
         },
     ]
 
     return (
         <>
-            <h2 className="subtitle">Organization Members</h2>
+            <h2 className="subtitle">Members</h2>
```

Comment:
Nit: Maybe best to use "Team Members" everywhere?

Reply:
IMO it's cleaner as just "Members", since the context is "Organization Settings"… Otherwise we still have the `Team` model.